### PR TITLE
terms and conditions drafts

### PR DESCRIPTION
**Pull request by lachellel**
Draft for general counsel review.  Not final pending reviews.

### CODE REVIEW

**Comment by lachellel** [comment]
@konklone quick review if you have time?

File: certificate-policy.md [hunk 1]
```diff
@@ -1786,122 +1786,134 @@ During the period in which a Subordinate CA issues Certificates, the Root CA sha
...
 ### 9.3.2 Information not within the scope of confidential information
+No stipulation.
 
 ### 9.3.3 Responsibility to protect confidential information
+A CA shall not disclose non-certificate information to any third party unless authorized by this policy, required by U.S. law, U.S. government rule or regulation, or order of a U.S. court of competent jurisdiction.  The contents of the archives maintained by CAs operating under this policy shall not be released except as required by this policy, required by U.S. law, U.S. government rule or regulation, or order of a U.S court of competent jurisdiction.  The FPKI Policy Authority must authenticate any request for release of information.
 
 ## 9.4 Privacy of personal information
 
 ### 9.4.1 Privacy plan
+CAs shall conduct a Privacy Impact Assessment, and implement and maintain any required Privacy Plans in accordance with the requirements of the Privacy Act of 1974, as amended.
```
Comment (lachellel) [inline certificate-policy.md]: Updating to be the Privacy Threshold Assessment - which may determine that a PIA and Privacy Plans are warranted, or not.

Comment (lachellel) [reply certificate-policy.md]: [great feedback from chief privacy officer]

File: certificate-policy.md [hunk 2]
```diff
@@ -1786,122 +1786,134 @@ During the period in which a Subordinate CA issues Certificates, the Root CA sha
...
+The CA is not required to provide any notice or obtain the consent of the Subscriber in order to release private information in accordance with other stipulations of section 9.4.
 
 ### 9.4.6 Disclosure pursuant to judicial or administrative process
+The FPKIPA or CAs shall not disclose private information to any third party unless authorized by this policy, required by law, government rule or regulation, or order of a court of competent jurisdiction.  Any request for release of information shall be processed according to 41 CFR 105-60.605.
 
 ### 9.4.7 Other information disclosure circumstances
+None. 
 
 ## 9.5 Intellectual property rights
+The FPKIPA and CAs will not knowingly violate intellectual property rights held by others.
```
Comment (konklone) [inline certificate-policy.md:1855]: What is this intended to address?

File: certificate-policy.md [hunk 3]
```diff
@@ -1915,39 +1927,44 @@ The review and update shall include any changes needed to address source require
...
 
 ### 9.12.2 Notification mechanism and period
+Proposed changes to this CP shall be distributed electronically to FPKIPA members and observers in accordance with the Charter and By-laws, and posted publicly for review by any interested party.  The FPKIPA shall make any subsequent changes publicly available within 30 days of approval (see Section 2.3). 
 
 ### 9.12.3 Circumstances under which OID shall be changed
+No stipulation. 
 
-## 9.13 Dispute resolution provisions
+## 9.13 Dispute resolution 
+The FPKIPA shall facilitate the resolution between entities when conflicts arise as a result of the use of certificates issued under this policy.  If a dispute is between U.S. Federal government entities, and the FPKIPA is unable to facilitate resolution, dispute resolution may be escalated to the Office of Management and Budget or to the U.S. Department of Justice, Office of Legal Counsel as necessary.
```
Comment (konklone) [inline certificate-policy.md]: I would say `White House Office of Management and Budget` to be explicit.

**Review by konklone — COMMENTED** [review]
Generally seems fine, though many aspects here aren't my area of expertise. The one thing that stuck out was the IP clause - we're issuing certificates for other people who give us their hostnames, it's not obvious to me where third party IP enters into the picture.

File: certificate-policy.md (hunk 2 above)
Comment (lachellel) [reply certificate-policy.md:1855]: this is a boilerplate.

**Pull request merged**